Cargo.toml

```diff
@@ -47,7 +47,7 @@ sha-1 = "^0.9.4"
 hex-literal = "^0.3.1"
 hex_fmt = "^0.3.0"
 command-extra = "^1.0.0"
-pacman = "^0.1.3"
+pacmanconf = "1.0.0"
```

Comment:
I prefer a caret (i.e. `^1.0.0`). I would have edited this file already if not for the need of updating `Cargo.lock`.

Reply:
Ah, I use `cargo add` so rarely touch the file myself. Will change it.

Reply:
I use `cargo add` as well, but with extra options:

```sh
cargo add --upgrade minor $PACKAGE
```